src/components/Content/Counter/Counter.js

```diff
@@ -0,0 +1,57 @@
+import './Counter.css';
+import React from "react";
+import Ingredient from './Ingredient/Ingredient';
+
+
+class Counter extends React.Component {
+
+    constructor () {
+        super();
+
+        this.output = this.output.bind(this)
```

Comment:
Try to move binding after `state`. It will organise your code better for readability